source/agora/test/Simple.d

```diff
@@ -41,4 +41,5 @@ unittest
     auto txes = genesisSpendable().map!(txb => txb.sign()).array();
     txes.each!(tx => node_1.putTransaction(tx));
     network.expectBlock(Height(1));
+
```

Comment:
:O

Reply:
I guess I changed my editor settings since last update on this file :-)

Reply:
Well it's an empty line, not trailing whitespace